Assets/Scripts/Commander.cs

```diff
@@ -4,8 +4,13 @@
 
 public class Commander
 {
-    public void SelectUnit(List<Unit> MyUnits)
+    public void AddUnit(Unit unit, List<Unit> myUnits)
     {
-        
+        myUnits.Add(unit);
+    }
+
+    public void SelectUnit(Unit unit)
+    {
+        unit.SelectMyself();
```

Comment:
아래 제가 수정 제안드린 대로 한다면 아래 처럼 코드가 바뀔 거에요.

```suggestion
        unit.Selected = true;
```